client/modules/User/pages/SignupView.jsx

```diff
@@ -27,9 +27,13 @@ function SignupView() {
             <SocialAuthButton service={SocialAuthButton.services.google} />
           </div>
           <p className="form__navigation-options">
-            By signing up, you agree to the p5.js Editor&apos;s{' '}
-            <Link to="/terms-of-use">Terms of Use</Link> and{' '}
-            <Link to="/privacy-policy">Privacy Policy</Link>.
+            <Trans
+              i18nKey="SignupView.Warning"
+              components={[
+                <Link to="/terms-of-use">Terms of use</Link>,
```

Comment:
Thanks for catching this!

We need to look over the i18next docs to make sure that we've got the right syntax for the `components`.  Having the phrase "Terms of use" here in English makes me nervous that the anchor text of the link won't get translated.  If you want to add the Hindi translations to this PR that would be helpful because the we would have a language that we can switch to in order to double-check the behavior.

Reply:
Okay I think that you have it right and the anchor text of the `<Link>` doesn't matter.  Weird.  https://react.i18next.com/latest/trans-component#alternative-usage-components-array

Reply:
Added Hindi Translation also.